showy.typ

```diff
@@ -72,36 +72,41 @@
         y: shadow.at("offset", default: 4pt)
       )
     }
-    wrap = (sbox) => move(
-      dx: shadow.offset.x,
-      dy: shadow.offset.y,
-      rect(
+    if not breakable {
+      wrap = (sbox) => move(
+        dx: shadow.offset.x,
+        dy: shadow.offset.y,
+        rect(
+          radius: frame.at("radius", default: 5pt),
+          fill:   shadow.at("color", default: luma(128)),
+          outset: 0pt,
+          inset:  0pt,
+          move(
+            dx:-1 * shadow.offset.x,
+            dy:-1 * shadow.offset.y,
+            sbox
+          )
+        )
+      )
+    } else {
+      wrap = (sbox) => block(
+        breakable: breakable,
         radius: frame.at("radius", default: 5pt),
         fill:   shadow.at("color", default: luma(128)),
-        outset: 0pt,
-        inset:  0pt,
-        move(
-          dx:-1 * shadow.offset.x,
-          dy:-1 * shadow.offset.y,
-          sbox
-        )
+        inset: (top:0pt, left: 0pt, right: shadow.offset.x, bottom: shadow.offset.y),
```

Comment:
I've found the Y-offset problem's solution!

```suggestion
        inset: (top: -shadow.offset.y, left: -shadow.offset.x, right: shadow.offset.x, bottom: shadow.offset.y),
```

Reply:
Cool, somehow couldn't get that to work. Your fix should remove the need to have different wrappers for breakable and non-breakable boxes altogether. 👍